packages/calypso-build/webpack.config.js

```diff
@@ -104,6 +104,9 @@ function getWebpackConfig(
 					ecma: 5,
 					safari10: true,
 					mangle: { reserved: [ '__', '_n', '_nx', '_x' ] },
+					format: {
+						comments: /translators:/i,
```

Comment:
This regexp could be a bit smarter, to catch the prefix only at the start of the comment. `/^\s*translators/i` should work.

Reply:
According to [docs](https://github.com/terser/terser#format-options), the `format.comments` option has a default value that preserves comments that contain a `@license` or `@preserve` jsdoc tag. Let's verify whether we're adding new rules in addition to the defaults, or overwriting them.

Reply:
Gutenberg has this line, so if we find improvements it would be nice to share upstream:

https://github.com/WordPress/gutenberg/blob/cefca421af1eeae1f9e8571e67ca795f9084261a/webpack.config.js#L78-L80
https://github.com/WordPress/gutenberg/blob/cefca421af1eeae1f9e8571e67ca795f9084261a/packages/scripts/config/webpack.config.js#L118-L120

Reply:
They use `output.comments` instead of `format.comments`.

Reply:
> They use `output.comments` instead of `format.comments`.

Seems like `output` has been replaced with `format` back in `terser@v5.0.0`, so that shouldn't really matter.

Reply:
I've been using a snippet like this:

```js
	// Keep the JS files parseable by make-pot
	webpackConfig.optimization.minimizer[ 0 ].options.terserOptions.output = {
		beautify: true,
		// Comments that Terser keeps by default, plus translators comments
		comments: /^\**!|@preserve|@license|@cc_on|translators:/i,
	};
```

Note that line breaks need to be present for the translator comments to be parsed correctly. Hence the beautify 😅